packages/dotnet/ReadMe/HarJsonTranslationLogics/RequestProcessor.cs

```diff
@@ -1,145 +1,154 @@
-﻿using Microsoft.AspNetCore.Http;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
 using System.IO;
 using System.Linq;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
 
 namespace ReadMe.HarJsonObjectModels
 {
   class RequestProcessor
   {
-    private readonly HttpRequest _request;
-    private readonly ConfigValues _configValues;
+    private readonly HttpRequest request;
+    private readonly ConfigValues configValues;
 
     public RequestProcessor(HttpRequest request, ConfigValues configValues)
     {
-      _request = request;
-      _configValues = configValues;
+      this.request = request;
+      this.configValues = configValues;
     }
 
     public async Task<Request> ProcessRequest()
     {
       Request requestObj = new Request();
-      requestObj.headers = GetHeaders();
-      requestObj.headersSize = GetHeadersSize();
-      requestObj.queryString = GetQueryStrings();
-      requestObj.cookies = GetCookies();
-      requestObj.method = _request.Method;
-      requestObj.url = _request.Scheme + "://" + _request.Host.Host + ":" + _request.Host.Port + "" + _request.Path;
```

Comment:
Removed the empty `"'` string from this line instead of replacing it with `string.Empty` because there's no need to concatenate `Port` and `Path` together with anything in between.